### ISSUE TITLE
BoundsError: attempt to access 7-element Array{Literate.Chunk,1} at index [0]

### ISSUE THREAD

**Issue by dpsanders** [fredrikekre/Literate.jl#68]
I'm getting the following error:
```jl
julia> Literate.notebook("juliacon_2019_interval_talk_25_july_2019.jl", "", execute=false)
[ Info: generating notebook from `~/Dropbox/talks/2019/juliacon_2019/juliacon_2019_intervals/juliacon_2019_interval_talk_25_july_2019.jl`
[ Info: not running on Travis, skipping links will not be correct.
ERROR: BoundsError: attempt to access 7-element Array{Literate.Chunk,1} at index [0]
Stacktrace:
 [1] getindex at ./array.jl:729 [inlined]
 [2] #parse#1(::Bool, ::Function, ::String) at /Users/dpsanders/.julia/packages/Literate/McVbf/src/Literate.jl:89
 [3] #parse at ./none:0 [inlined]
 [4] #notebook#11(::typeof(identity), ::typeof(identity), ::Bool, ::Bool, ::Bool, ::String, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::String, ::String) at /Users/dpsanders/.julia/packages/Literate/McVbf/src/Literate.jl:441
 [5] (::getfield(Literate, Symbol("#kw##notebook")))(::NamedTuple{(:execute,),Tuple{Bool}}, ::typeof(Literate.notebook), ::String, ::String) at ./none:0
 [6] top-level scope at none:0
```

The input file is
```jl
# # Constraints

# - **Constraint**: relation between

# - **Constraint satisfaction problem**
        > Find the **feasible set** of values satisfying all constraints

# - E.g.
# \begin{align}
#  x + y \le 4
#  x + 3y \le 9
#  -4 \le x, y \le 4
# \end{align}

using IntervalArithmetic, IntervalConstraintProgramming

C1 = @constraint x + y ≤ 4
C2 = @constraint x + 3y ≤ 9
C = C1 ∩ C2

X = IntervalBox(-4..4, 2)

paving = pave(C, X, 0.1)

using Plots
plot(paving.inner, aspect_ratio=1)
plot!(paving.boundary)
```

Julia 1.1, Literate 1.1.0

**Comment by dpsanders** [comment]
Ah apologies, the problem was the un-commented line `     > Find the feasible set`.

But maybe a better error message would be possible.

**Comment by fredrikekre** [comment]
Yea, could be a better error message, and maybe show the block that failed.

**Comment by dpsanders** [comment]
And suggest to add `#` at the start of each line.

**Comment by fredrikekre** [comment]
I seem to have accidentally fixed this issue, probably by 36e8c210478a8be83ce0b2ce961ecd5c1abc8b45
